src/Cache.js

```diff
@@ -313,6 +277,8 @@ CM.Cache.RemakeSellForChoEgg = function() {
 		}
 		sellTotal += goodsVal * Game.cookiesPsRawHighest;
 	}
+	// Compute cookies earned by selling all buildings with optimal auras (ES + RB)
+	sellTotal += CM.Sim.SellBuildingsForChoEgg();
```

Comment:
I've moved the buildings selling value after the stock market selling to be consistent with the way you do it for real (as you can't sell all your stock market goods if you don't have any banks)